app/lib/lnd/neutrino.js

```diff
@@ -55,6 +80,17 @@ class Neutrino extends EventEmitter {
       `${this.lndConfig.alias ? `--alias=${this.lndConfig.alias}` : ''}`
     ]
 
+    switch (this.lndConfig.network) {
```

Comment:
Why did you opt for a `switch` statement with only 2 options? Are we expecting this conditional to grow? There is also no default clause here

Reply:
Good question. Actually at the time I had in my mind about supporting multiple different currencies but even then each one would probably only have testnet and mannet support. So I'll update this to use if/else.